dsa/beginner/searching-techniques.md

```diff
@@ -0,0 +1,167 @@
+# Searching Techniques:
+
+Searching is a technique that which helps to find the place of a given element. Any Search is said to be unsuccessful or successful depending upon whether the element being searched is found or not.
+
+Majorly, Searching techniques are of two types:
+
+ - Linear Search
+ - Binary Search
+
+# Linear Search:
+Linear search is a very simple search algorithm.In this type of search, a sequential search is made over all elements one by one.Every item is checked and if match is found then it will be returned true otherwise, the search continues till the end of the data.
+
+It has a time complexity of O(N)
+
+## Algorithm:
+Linear search(array A, value x)
+
+Step1 : Set an integer i to 1
+
+Step2: If i>n then , go to step7 (where n is the size of array)
```

Comment:
use code tag or <code>`</code> 

for example

**bad practices** 

```md
>
```

**Good practice**

```md

`>`

```